src/webhooks/options-automator/options-automator.ts

```diff
@@ -26,15 +26,49 @@ export async function sendOptionAutomatorUpdatesToDataDog(
   message: OptionsAutomatorResponse,
   timestamp: number
 ) {
-  // Not sure how detailed we want the text message to be, but this should be fine initially
-  const params: v1.EventCreateRequest = {
-    title: 'Options Automator Update',
-    text: JSON.stringify(message),
-    alertType: 'info',
-    dateHappened: timestamp,
-    tags: [`sentry_region:${message.region}`],
+  const formatRegionTag = (region: string): string => {
+    if (region === 'us' || region === 'de') {
+      return `sentry_region:${region}`;
+    } else {
+      return `sentry_region:st-${region}`;
+    }
+  };
+
+  const formatAlertType = (optionType: string): v1.EventAlertType => {
+    return optionType === 'updated_options' ||
+      optionType === 'set_options' ||
+      optionType === 'unset_options'
+      ? 'success'
+      : 'error';
   };
-  await DATADOG_API_INSTANCE.createEvent({ body: params });
+
+  for (const optionType in message) {
+    if (optionType === 'region') continue;
+    for (const option of message[optionType]) {
+      const text = {
+        change: optionType,
+        option: option,
+      };
+
+      const region = formatRegionTag(message.region);
+
+      const alertType = formatAlertType(optionType);
+
+      const params: v1.EventCreateRequest = {
+        title: 'Options Automator Update',
+        text: JSON.stringify(text),
```

Comment:
I do think it would be helpful to use a more consise paragraph for `text`, I initially just used `JSON.stringify` because I didn't have enough time to write a custom message. It seems like since now an event is created for every option that is updated, it feels right to clean this up. 

Not blocking here, but just wanted to raise it up as potentially a TODO for the future

Reply:
can do something similar to what gets sent to slack?

Reply:
Yeah this can be a potential todo for the future. It would be a bit of refactoring.

Reply:
can we create a GitHub Issue for this then? Just so we can track this and include a comment

`// TODO(getsentry/eng-pipes#144): Refactor Text Message`